internal/raft/append_entries_test.go

```diff
@@ -0,0 +1,77 @@
+package raft
+
+import (
+	"net"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+	"github.com/tomarrell/lbadd/internal/network"
+	"github.com/tomarrell/lbadd/internal/raft/cluster"
+	"github.com/tomarrell/lbadd/internal/raft/message"
+)
+
+func TestAppendEntries(t *testing.T) {
```

Comment:
`Test_AppendEntries`*

Reply:
according to go convention of naming Test function,
The function name should be prefixed with Test , followed by a capitalised word, and should have the signature func TestXxx(t *testing.T).